clang-tools-extra/clang-tidy/readability/ReadabilityTidyModule.cpp

```diff
@@ -119,6 +120,8 @@ class ReadabilityModule : public ClangTidyModule {
         "readability-redundant-preprocessor");
     CheckFactories.registerCheck<ReferenceToConstructedTemporaryCheck>(
         "readability-reference-to-constructed-temporary");
+    CheckFactories.registerCheck<ReturnExpressionInVoidFunctionCheck>(
+        "readability-return-expression-in-void-function");
```

Comment:
I'm not sure about this name, if this is about return argument, or just an return

Reply:
What about the succinct `readability-void-return`?

Reply:
Consider: readability-return-void-value or readability-void-return-statement or readability-avoid-return-with-void-value.

Reply:
`readability-avoid-return-with-void-value` sounds good.